lib/types.ts

```diff
@@ -920,7 +920,9 @@ export class FromStream implements IOOperation {
         return this.ioId
     }
     async toOutput(buffer: ArrayBuffer, collector: Object): Promise<any> {
```

Comment:
shouldn't this change from
`async toOutput(buffer: ArrayBuffer, collector: Object): Promise<any>`
to
`toOutput(buffer: ArrayBuffer, collector: Object): void`

Reply:
yeah.